src/main/java/seedu/address/logic/parser/expense/AddExpenseCommandParser.java

```diff
@@ -43,7 +43,7 @@ public AddExpenseCommand parse(String args) throws ParseException {
         }
 
         Description description = ParserUtil.parseDescription(argMultimap.getValue(PREFIX_DESCRIPTION).get());
-        IsFixed isFixed = ParserUtil.parseIsFixed(argMultimap.getValue(PREFIX_ISFIXED).get());
+        IsFixed isFixed = ParserUtil.parseIsFixed(argMultimap.getValue(PREFIX_ISFIXED).get().charAt(0));
```

Comment:
With this implementation of the parsing of user input, what happens if the user inputs something like:
`addexp ... f/for now no` OR  `addexp ... f/think no`. 

Although in an actual use scenario this is very unlikely to, or will never happen, but would the parser still allow the creation of the expense object in this scenario?

Reply:
Noted, I have changed it from a character to a String. Thanks for pointing out!